dependencies/FindFastJet.cmake

```diff
@@ -94,7 +94,7 @@ find_package_handle_standard_args(${PKGNAME}
 
 # if everything was found, assemble target with dependencies
 if(${${PKGNAME}_FOUND})
-  add_library(${PKGNAME}::${PKGNAME} IMPORTED INTERFACE GLOBAL)
+  add_library(${PKGNAME}::${PKGNAME} IMPORTED INTERFACE GLOBAL ../Framework/Core/src/SpyService.cxx ../Framework/Core/src/SpyService.h ../Framework/Core/src/SpyServiceHelpers.cxx ../Framework/Core/src/SpyServiceHelpers.h)
```

Comment:
Why do you need this? You can probably just modify Framework/Core/CMakeLists.txt